pkg/stanza/operator/parser/csv/csv.go

```diff
@@ -54,6 +54,7 @@ type Config struct {
 	HeaderAttribute string `mapstructure:"header_attribute" json:"header_attribute" yaml:"header_attribute"`
 	FieldDelimiter  string `mapstructure:"delimiter" json:"delimiter,omitempty" yaml:"delimiter,omitempty"`
 	LazyQuotes      bool   `mapstructure:"lazy_quotes" json:"lazy_quotes,omitempty" yaml:"lazy_quotes,omitempty"`
+	IgnoreQuotes    bool   `mapstructure:"ignore_quotes" json:"ignore_quotes,omitempty" yaml:"ignore_quotes,omitempty"`
```

Comment:
This setting bypasses the use of `LazyQuotes`. Should we throw a build error if both are set, just to avoid ambiguous behavior in that case? Are there any other settings that are ignored when this is set?

Reply:
Good call out; I've made it so that this is a build error.

All the other settings are still used when `IgnoreQuotes` is true, so we shouldn't need to do this for any of the other fields.